src/momento/responses/data/sorted_set/fetch.py

```diff
@@ -0,0 +1,68 @@
+from __future__ import annotations
+
+from abc import ABC
+from dataclasses import dataclass
+
+from ...mixins import ErrorResponseMixin
+from ...response import CacheResponse
+
+
+class CacheSortedSetFetchResponse(CacheResponse):
+    """Response type for a `sorted_set_fetch` request.
+
+    Its subtypes are:
+    - `CacheSortedSetFetch.Hit`
+    - `CacheSortedSetFetch.Miss`
+    - `CacheSortedSetFetch.Error`
+
+    See `CacheClient` for how to work with responses.
+    """
+
+
+class CacheSortedSetFetch(ABC):
+    """Groups all `CacheSortedSetFetchResponse` derived types under a common namespace."""
+
+    @dataclass
+    class Hit(CacheSortedSetFetchResponse):
+        """Indicates the sorted set exists and its values were fetched."""
+
+        value_dictionary_bytes: dict[bytes, float]
+        """The values for the fetched sorted set, as bytes to their float scores."""
+
+        @property
+        def value_dictionary_string(self) -> dict[str, float]:
+            """The values for the fetched sorted set, as utf-8 encoded strings and their int scores.
+
+            Returns:
+                dict[str, float]
+            """
+            return {key.decode("utf-8"): value for key, value in self.value_dictionary_bytes.items()}
+
+        @property
+        def value_list_bytes(self) -> list[tuple[bytes, float]]:
+            """The values for the fetched sorted set, as a list of bytes values to int score tuples.
+
+            Returns:
+                list[tuple[bytes, float]]
+            """
+            return [(key, value) for key, value in self.value_dictionary_bytes.items()]
+
+        @property
+        def value_list_string(self) -> list[tuple[str, float]]:
+            """The values for the fetched sorted set, as a list of utf-8 encoded string values to int score tuples.
+
+            Returns:
+                list[tuple[str, float]]
+            """
+            return [(key.decode("utf-8"), value) for key, value in self.value_dictionary_bytes.items()]
```

Comment:
Since the order is lost when you store as a dictionary, could you: store the list instead of the dictionary as the response payload, then build the dictionaries on the fly? See [here](https://github.com/momentohq/client-sdk-python/blob/main/src/momento/responses/data/dictionary/get_fields.py) for an example where we want both the order preserved (list) and dictionary accessors.